.github/workflows/ci.yml

```diff
@@ -16,51 +14,24 @@ permissions:
 
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # See <https://julialang-s3.julialang.org/bin/versions.json> for available Julia versions
-        include:
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.10"
-          - arch: "x64"
-            os: "windows-latest"
-            version: "1.10"
-          - arch: "x64"
-            os: "macOS-latest"
-            version: "1.10"
-          - arch: "x86"
-            os: "ubuntu-latest"
-            version: "1.10"
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.9"
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.8"
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.7"
-          - arch: "x64"
-            os: "ubuntu-latest"
-            version: "1.6"
-          # - arch: "x64"
-          #   os: "ubuntu-latest"
-          #   version: "nightly"
+        julia-version: ['lts', '1']
+        julia-arch: [x64, x86]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        exclude:
+          - os: macOS-latest
```

Comment:
`macos-latest` is now an ARM system. We should probably be explicitly about version numbers here and test on both `macos-13` (x86-64) and `macos-14` (arm64). The interesting thing here is that we can actually test on an ARM system!

It would probably also be good to be explicit about the Ubuntu and Windows versions.